src/app/services/technical-record/technical-record.service.ts

```diff
@@ -37,6 +37,16 @@ export class TechnicalRecordService {
     return this.getVehicleTechRecordModels(vin, SEARCH_TYPES.VIN);
   }
 
+  isUnique(valueToCheck: string, searchType: SEARCH_TYPES) {
```

Comment:
This breaks the cohesion of the service. The service follows the order of get, create, update, delete/archive, and miscellaneous. So, this should go under the primary methods. Can you move this after `archiveTechnicalRecord()`, please?

Reply:
No longer valid after your last commit.